tutorials/scripting/gdscript/gdscript_documentation_comments.rst

```diff
@@ -185,6 +177,9 @@ be used in the documentation. Here's the list of available tags:
 +---------------------------+--------------------------------+-----------------------------------+---------------------------------------------------+
 | [signal Class.signalname] | Link to another class's signal | Emit [signal Node.renamed].       | Emit :ref:`renamed <class_node_signal_renamed>`.  |
 +---------------------------+--------------------------------+-----------------------------------+---------------------------------------------------+
+| [br]                      | Line break                     | | Line 1.[br]                     | | Line 1.                                         |
+|                           |                                | | Line 2.                         | | Line 2.                                         |
```

Comment:
There seems to be extraneous "|` before "Line 1" and "Line 2" on both columns.

Reply:
These `|` are not an mistake, they are needed to add a line break in a table cell. Compare the difference in the result: [current](https://github.com/godotengine/godot-docs/blob/ed84099d437b1dc63692d4aeff0d188df34c4efd/tutorials/scripting/gdscript/gdscript_documentation_comments.rst) and [without extra pipes](https://github.com/dalexeev/godot-docs/blob/example-for-pipes/tutorials/scripting/gdscript/gdscript_documentation_comments.rst).

Should the line break in the Usage column be removed to make the `[br]` effect more pronounced?

Reply:
That makes sense, I wasn't aware of that rST syntax construct. Looks good to me as it is now :slightly_smiling_face: